### ISSUE TITLE
Footnotes aren't working in top sections titles

### ISSUE THREAD

**Issue by manuel489** [metanorma/metanorma-standoc#212]
Footnotes don't work properly when they are set in top sections titles, i.e. the ones that begin with `==`. They work properly on subsections though.

For example, this markup:

````
== Section footnote:[(Footnotes in sections aren't working)]
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse velit est, ullamcorper id ligula sit amet, volutpat elementum lorem. Curabitur efficitur convallis velit, in dictum risus efficitur vitae. In ut convallis est.

=== Subsection footnote:[Footnotes in subsections are working]
Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Pellentesque ultricies quam eu urna efficitur, sed interdum sapien rutrum. Nam quis malesuada metus. Nulla sit amet odio sit amet est bibendum venenatis. Nulla a condimentum metus.
````
Results in this output:
![footnote-issue-capture](https://user-images.githubusercontent.com/33627611/74359724-2dfbcb00-4d9a-11ea-91f2-36457f72d535.JPG)

Sample file:
[footnote-issue.zip](https://github.com/metanorma/metanorma-standoc/files/4193787/footnote-issue.zip)

**Comment by opoudjis** [comment]
This is an issue that I had fixed once before, but it may well have come back with the upgrade in standoc; section titles are processed in a different ordering from other content.

**Comment by opoudjis** [comment]
No, https://github.com/metanorma/metanorma-standoc/issues/13 was about footnote numbering.

**Comment by opoudjis** [comment]
In isodoc, need to parse all section titles; was just dumping the content of the topmost clause names.

**Comment by opoudjis** [comment]
If an override title is used in isodoc (mandatory title), it will not have a footnote, because it is ignoring what is there in the source. Will note as gotcha.
